docs/meshstack.cloudfoundry.index.md

```diff
@@ -19,6 +19,24 @@ The tenant [replication](./meshcloud.tenant.md) ensures spaces and orgs are crea
 
 ![Cloud Foundry Architecture](assets/cf-architecture.svg)
 
+## CloudFoundry Versions
+
+In general all CloudFoundry versions that provide the V2 API for the following endpoints are supported:
+
+* /v2/organizations
+* /v2/quota_definitions
+* /v2/space_quota_definitions
+* /v2/spaces
+* /v2/users
+* /v2/apps
+* /v2/private_domains
+* /v2/domains
+
+Integration with the following Cloud Foundry API Versions are additionally tested successfully:
+
+* 2.155.0
```

Comment:
d: maybe put this first like

meshcloud officially validates compatibility with the following API versions. In general all version that provide V2 API for...